packages/immutable/immutable.0.0.6/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Dave Bordoley <bordoley>"
```

Comment:
Could you please include a full email address in the `maintainer` field, @bordoley?